ajax/libs/simplebar/3.1.1/simplebar.js

```diff
@@ -172,7 +172,7 @@
     return store[key] || (store[key] = value !== undefined ? value : {});
   })('versions', []).push({
     version: _core.version,
-    mode: 'global',
+    mode: _library ? 'pure' : 'global',
```

Comment:
This file was added from the previous commit, which also claims it's the latest, it's weird to be updated again here, is there anything I missed here?

Reply:
@Grsmto would you please take a look? Thanks.

Reply:
I have no idea why this was updated.
To be honest I spent so so so much time on this PR, I think I'm about to give up. Every time I need to update something I spend at least an hour with the process in here :(

Reply:
I double checked and the file are 1:1 with the npm version. It was a mistake from a while back. As I'm hard pushing it looks like it's from the latest commit but it's not.

However of course in between I released a new version of SimpleBar (3.1.3) so I need to add that version as well before this can be merged.......

Reply:
I know it's a little bit hard working on a huge repository, and thanks for your time on it! To provide the best integrity and reliable service, we just need to make sure the files are correct, that's why I take a deeper look into the pull request. I'll take another check here soon to see if everything is okay.

BTW, not sure if you enabled sparse checkout, which will let you not spend so much time on it.

Reply:
So it should be at b25f999ec3787872e45ba531fe1467d8b60160c4 or bbb10c7a3af24a7b1d505851ae14e323efc2826a, one of them contains the wrong files, I'll check that.

Reply:
Looks like the later commit is correct, the previous isn't.

As the wrong history here is no needed in the repositoy, do you mind to do a little final revise to fix that? It'll be even better to mention the exact version in the commit message, instead of the "latest" version, as when you need to reace the update history here(Fix v3.1.1 in the previous commit, or, simple suqash that two commit of adding new versions) and see many of the latest, the more precise the more clear.

Thanks again.

Reply:
Squashed everything under 1 commit which was the original commit message.

Reply:
Actually, the preferred form here is to separate the config part and the assets in the individual commit, the reason why I mention the squash is that the previous one of the assets adding commit was containing wrong content, which was fixed in the later commit, then a suqash can help you make it disappear in that situation, anyway, I'll try to make this one easier, get it merged ASAP, thanks agin!